# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/smartmontools/noinst-init.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2003-04-23 15:56:17.000000000 +0200
+++ ./Makefile	2003-07-02 00:33:06.000000000 +0200
@@ -118,7 +118,6 @@
 	rm -f $(DESTDIR)/usr/share/man/man8/smartd.8
 	$(INSTALL_PROGRAM) -D smartctl         $(DESTDIR)/usr/sbin/smartctl
 	$(INSTALL_PROGRAM) -D smartd           $(DESTDIR)/usr/sbin/smartd
-	$(INSTALL_PROGRAM) -D smartd.initd     $(DESTDIR)/etc/rc.d/init.d/smartd
 	$(INSTALL_DATA) -D smartctl.8.gz    $(DESTDIR)/usr/share/man/man8/smartctl.8.gz
 	$(INSTALL_DATA) -D smartd.8.gz      $(DESTDIR)/usr/share/man/man8/smartd.8.gz
 	$(INSTALL_DATA) -D smartd.conf.5.gz $(DESTDIR)/usr/share/man/man5/smartd.conf.5.gz
